skietbaan_fe/src/components/CreateComp.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { Jumbotron, Form, Button } from 'react-bootstrap';
+import propTypes from 'prop-types';
+import { connect } from 'react-redux';
+import '../scss/createcomp.css';
+import { createcomp } from '../actions/competition.action';
+class CreateComp extends Component {
+	constructor() {
+		super();
+		this.state = { compTitle: '', compStatus: '' };
+		//binding the onChange method to this commponents
+		this.onChange = this.onChange.bind(this);
+		this.onSubmit = this.onSubmit.bind(this);
+	}
+	//------------------------------------------------------------------------
+	/** A method that detects the change in the change in thw textfield */
+	onChange(e) {
```

Comment:
e is easy to easy and faster but make it more meaningul.
keep it simple by replacing it with event in your upcoming event manipulation

Reply:
I here you, but the onChange method caters for other Text inputs that may be included in the future

Reply:
Cool